### PR TITLE
RDCC-1805 Remove foreign key constraint from audit table

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
RDCC-1805

### Change description ###
RDCC-1805 : Remove foreign key constraint from audit table and also rename response table to audit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -353,15 +354,15 @@ dependencies {
         exclude group: "com.sun.xml.bind", module: "jaxb-osgi"
     }
 
-    testCompile(group: 'org.yaml', name: 'snakeyaml', version: '1.25') {
+    testCompile(group: 'org.yaml', name: 'snakeyaml', version: '1.26') {
```
Comment (abhijitdiwan) [inline build.gradle:357]: i think we can remove testCompile(group: 'org.yaml' as it is  compile group:"org.yaml", name: "snakeyaml", version:"1.26" and all will get compile.same for me also in my pr :)

Comment (shreedhar-hmcts) [reply build.gradle]: Done

**Review by KOTLAPRA23 — COMMENTED** [review]
deleteUserProfiles method in DeleteUserProfileServiceImpl modify the comment.

**Comment by shreedhar-hmcts** [comment]
All comments taken care.

**Comment by shreedhar-hmcts** [comment]
> deleteUserProfiles method in DeleteUserProfileServiceImpl modify the comment.

done

**Review by KOTLAPRA23 — APPROVED** [review]

**Comment by shreedhar-hmcts** [comment]
Closing this since Table needs to be changed. Requires new PR for table change in flyway.

**Pull request closed without merging**